site/content/introduction/showcase.md

```diff
@@ -5,7 +5,7 @@ draft: false
 weight: 3
 ---
 
-NoSQLBench has enjoyed a history of unique innovation, driven by the vision of its users and
+NoSQLBench has enjoyed a history of unique innovation: driven by the vision of its users and
```

Comment:
Not sure about this change. I think the comma works better.

Reply:
Agreed. Not actually sure why I did that, must've been a mistake